wdl/src/main/scala/wdl/WdlTask.scala

```diff
@@ -109,93 +111,27 @@ case class WdlTask(name: String,
     * @param valueMapper Optional WomValue => WomValue function that is called on the result of each expression
     *                    evaluation (i.e. evaluation of $${...} blocks).
     * @return String instantiation of the command
+    *
+    * TODO WOM this method should go away in the #2944 purge
```

Comment:
If the test is only hitting this function, and this function is only used in that test, can we just chuck them both out immediateyly?